analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -190,34 +191,36 @@ public void Verify_UnchangedFiles(string unchangedFileName, bool expectedProtobu
 #if NET
 
         [TestMethod]
-        public void Verify_Razor() =>
+        public void Verify_Razor()
+        {
+            using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+
             // Currently only the symbols from .cs files are computed since .razor support is not yet implemented.
             CreateBuilder(ProjectType.Product, "Razor.razor", "ToDo.cs")
                 .WithConcurrentAnalysis(false)
                 .VerifyUtilityAnalyzer<SymbolReferenceInfo>(symbols =>
-                {
-                    symbols.Should().ContainSingle();
-                    symbols[0].FilePath.Should().EndWith("ToDo.cs");
-                    symbols[0].Reference
-                              .Select(x => x.Declaration)
-                              .Should()
-                              .BeEquivalentTo(new[]
-                                  {
-                                      new TextRange { StartLine = 3, EndLine = 3, StartOffset = 13, EndOffset = 17 },
-                                      new TextRange { StartLine = 5, EndLine = 5, StartOffset = 19, EndOffset = 24 },
-                                      new TextRange { StartLine = 6, EndLine = 6, StartOffset = 16, EndOffset = 22 }
-                                  });
-                });
+                    {
+                        var orderedSymbols = symbols.OrderBy(x => x.FilePath).ToArray();
+                        orderedSymbols.Select(x => Path.GetFileName(x.FilePath)).Should().Contain("_Imports.razor", "Razor.razor", "ToDo.cs");
+                        orderedSymbols[0].FilePath.Should().EndWith("_Imports.razor");
+                        orderedSymbols[1].FilePath.Should().EndWith("Razor.razor");
+
+                        VerifyReferences(orderedSymbols[1].Reference, 9, 13, 4, 6, 20); // currentCount
```

Comment:
Can we align the in-line comments vertically? That should positively affects readability.